src/app/organisation/assignrole/get.controller.js

```diff
@@ -1,8 +1,13 @@
 
 const CookieModel = require('../../../common/models/Cookie.class');
-const roles = require('../../../common/seeddata/egar_user_roles.json');
+let roles = require('../../../common/seeddata/egar_user_roles.json');
 
 module.exports = (req, res) => {
   const cookie = new CookieModel(req);
+ 
+  if(cookie.getUserRole() !== 'Admin'){
+    roles = roles.filter(role => role.id !== '0');
```

Comment:
Making this more readable. "Admin" more readable than "0". 

```suggestion
    roles = roles.filter(role => role.name !== 'Admin');
```